_minutes/2024-09-24-wecg-tpac.md

```diff
@@ -0,0 +1,466 @@
+   * [devlin] We're largely happy with Chrome's current behavior of keeping web data in spanning mode, destroying it in split. Seems like Firefox needs to decide what the extension behavior in general when “never remember history” is enabled.
+   * [rob] Never remember is permanent private mode, so destroying data is appropriate. Any appetite for providing a way to get a persistent indexedDB session?
+   * [devlin] No.
+   * [tomislav] I think Devlin is saying Firefox is free to handle this however it wants given the lack of consistency.
+   * Resolution: None!
+   * **Action item**: Devlin to add rationale for current Chrome behavior [[done](https://github.com/w3c/webextensions/issues/534#issuecomment-2374769783)]
+   * **Action item**: Firefox to decide how they want to handle IndexedDB in “never remember” mode.
+ * [Issue 570](https://github.com/w3c/webextensions/issues/570): Inconsistency: dns.resolve()
+   * [oliver] Only action item here was to check with the Chrome security team to see if we could expose our existing `dns` API implementation. I haven't done that and will try to do it in the future.
+   * **Action item**: Oliver to follow up.
+ * [Issue 507](https://github.com/w3c/webextensions/issues/507): Support $schema in manifest.json
+   * [devlin] What's the request?
+   * [carlos] Allow extensions to use a “$schema” key in the manifest.
+   * [oliver] Doesn't throw an error, but does return a warning.
+   * [david] There seems to be some confusion. Users reported that it threw an error when uploading but later stopped returning an error.
+   * [carlos] There are more keys that could raise warnings. Like browser_specific_settings.
+   * [oliver] Browser specific settings is a feature we don't support, but has some value elsewhere. “$schema”
+   * [devlin] we warn on browser_specific_settings because we don't support it. Developers try to specify it expecting it to … Sorry, was thinking browser overrides, not browser specific settings.
+   * [rob] Would you consider not warning on browser_specific_settings?
+   * [timothy] Safari also uses it.
+   * [devlin] Open to it.
+   * [devlin] Back to this issue, hesitant to support arbitrary fields used by some random tools. Worried about proliferation of everyone's pet keys.
+   * [timothy] $schema is a standard.
+   * [tomislav] Agreed with reasoning.
+   * [devlin] If this is used by a large amount of the community, I'm in favor of ignoring it.
+   * [rob] So we are committing to maintaining a list of intentionally recognized properties even if ignored, starting with browser_specific_settings and $schema.
+   * [devlin] Yes.
+   * [rob] There is already a Chromium issue for this at https://issues.chromium.org/issues/40196501.
+   * **Resolution**: Agreement that $schema should not be warned about.
+   * **Resolution**: Chrome to intentionally ignore browser_specific_settings and $schema instead of warning.
+     * [rob] Note: Oliver is starting to work on a patch to do this.
```

Comment:
```suggestion
     * [rob] Note: This is now implemented in Chrome: https://github.com/w3c/webextensions/issues/507#issuecomment-2384073091
```

Reply:
While this is true now, this was not the case at the time that the comment was made.

Reply:
There are enough retrospective notes in this doc that I'm not concerned about that.